tests/ci_folder_parsing.py

```diff
@@ -46,8 +46,8 @@
 _token = args.token
 _number = args.number
 
-if _number and not _class or not _os:
-    sys.exit("Unittest number provided without specifying supporting arguments:"
+if _number and not _class or _number and not _os:
```

Comment:
or _number and not _os:

i dont understand this use case, can you explain it to me?  It should be both, class and os  no?